pom.xml

```diff
@@ -1,204 +1,263 @@
+                                </resources>
+                            </configuration>
+                        </execution>
+                    </executions>
+                </plugin>
+                <plugin>
+                    <artifactId>maven-war-plugin</artifactId>
+                    <configuration>
+                        <webResources>
+                            <resource>
+                                <directory>${project.basedir}/../terasoluna-gfw-functionaltest-env/${profile.directory}/webapp/WEB-INF</directory>
+                                <targetPath>WEB-INF</targetPath>
+                                <filtering>true</filtering>
+                                <excludes>
+                                    <exclude>**/.gitkeep</exclude>
+                                </excludes>
+                            </resource>
+                        </webResources>
+                    </configuration>
+                </plugin>
+                <plugin>
+                    <groupId>org.codehaus.mojo</groupId>
+                    <artifactId>sql-maven-plugin</artifactId>
+                    <version>${org.codehaus.mojo.sql-maven-plugin.version}</version>
+                </plugin>
+            </plugins>
+        </pluginManagement>
+    </build>
+    <dependencyManagement>
+        <dependencies>
+            <dependency>
+                <groupId>${project.groupId}</groupId>
+                <artifactId>terasoluna-gfw-functionaltest-domain</artifactId>
+                <version>${project.version}</version>
+            </dependency>
+            <dependency>
+                <groupId>${project.groupId}</groupId>
+                <artifactId>terasoluna-gfw-functionaltest-web</artifactId>
+                <version>${project.version}</version>
+            </dependency>
+            <dependency>
+                <groupId>${project.groupId}</groupId>
+                <artifactId>terasoluna-gfw-functionaltest-env</artifactId>
+                <version>${project.version}</version>
+            </dependency>
+
+            <!-- == Begin Database == -->
+            <dependency>
+                <groupId>org.postgresql</groupId>
+                <artifactId>postgresql</artifactId>
+                <version>${postgresql.version}</version>
+            </dependency>
+            <dependency>
+                <groupId>com.oracle</groupId>
+                <artifactId>ojdbc7</artifactId>
+                <version>${ojdbc7.version}</version>
+            </dependency>
+            <dependency>
+                <!-- db2jcc4.jar must be pre-installed in local repository. --> 
+                <groupId>com.ibm</groupId>
+                <artifactId>db2jcc4</artifactId>
+                <version>${db2jcc4.version}</version>
+            </dependency>
+            <!-- == End Database == -->
+
+        </dependencies>
+    </dependencyManagement>
+    <properties>
+        <profile.directory>src/main</profile.directory>
```

Comment:
I was wondering whether to use `env`, because it has been reserved in the environment variable.
I think... for example, how about using `project.env.*` or `env-project.*` is ?